Sources/TuistEnvKit/Installer/BuildCopier.swift

```diff
@@ -12,6 +12,7 @@ class BuildCopier: BuildCopying {
     /// Files that should be copied (if they exist).
     static let files: [String] = [
         "tuist",
+        Constants.templatesDirectoryName,
```

Comment:
If the user tries to install a version of Tuist that doesn't include `/Templates` the `copy` method below will fail because `/bin/cp` will exit unsuccessfully. You'll need to handle that scenario gracefully:

```swift
class BuildCopier {
  static let files: [(path: String, optional Bool)] = [
    (path: Constants.templatesDirectoryName, optional: true)
  ]

}
```